pkg/controller/atlasdeployment/advanced_deployment.go

```diff
@@ -107,7 +107,11 @@ func advancedDeploymentIdle(ctx *workflow.Context, project *mdbv1.AtlasProject,
 		return atlasDeploymentAsAtlas, workflow.Terminate(workflow.Internal, err.Error())
 	}
 
-	atlasDeploymentAsAtlas, _, err = ctx.Client.AdvancedClusters.Update(context.Background(), project.Status.ID, deployment.Spec.AdvancedDeploymentSpec.Name, deploymentAsAtlas)
+	// TODO: Potential bug with disabling autoscaling if it was previously enabled
+	// j, _ := json.MarshalIndent(deploymentAsAtlas, "", " ")
+	// fmt.Println("DEBUG AFTER REGION SYNC", string(j))
```

Comment:
Should we not remove this in the final version?

Reply:
Fixed